tests_thien/pages/dashboard/login_page.py

```diff
@@ -0,0 +1,35 @@
+from selenpy.element.text_box import TextBox
+from selenpy.support.conditions import have
+from selenpy.support import browser
+from selenpy.element.base_element import BaseElement
+from selenpy.element.combo_box import ComboBox
+from tests_thien.utilities import constant
+
+class DashBoardLoginPage():
+
+    _drp_repository = ComboBox("name=repository")
+    _txt_username = TextBox("id=username")
+    _txt_password = TextBox("id=password")
+    _btn_login = BaseElement("class=btn-login")
+    
+    def __init__(self):
+        pass
+    
+    def open(self):
+        browser.open_url(constant.DA_URL)
+        browser.wait_until(have.title(constant.DA_TITLE))
+        
+    def login(self,repository,username,password):
+        self._drp_repository.select_by_visible_text(repository)
+        self._txt_username.send_keys(username)
+        self._txt_password.send_keys(password)
+        self._btn_login.click()
+        
+    def check_error_popup_displays(self):
+        try:
+            return browser.get_alert().text == "Username or password is invalid"
```

Comment:
Don't hard "Username or password is invalid".

Reply:
updated with a msg variable